map.go

```diff
@@ -44,6 +44,10 @@ type MapOptions struct {
 
 	// AlwaysCleanup - Overrides the clean up type given to the manager. See CleanupType for more.
 	AlwaysCleanup bool
+
+	// DumpHandler - Callback function called when manager.Dump() is called
+	// and dump the current state (human readable)
+	DumpHandler func(currentMap *Map, manager *Manager) string
```

Comment:
Should we return `(interface, error)` so that each dump handler can decide what to export when dumping a map ? I'm worried forcing a string will limit this feature to debugging purposes.

Reply:
(the error would stop the loop)

Reply:
we have already a filter that call DumpHandler() if the map name is selected
https://github.com/DataDog/datadog-agent/blob/main/pkg/network/tracer/tracer.go#L580

I think we can still keep simple as-is the handler can still return an empty string if for some reason/filter want to skip some manager and/or map